examples/python_3dunet/README.md

```diff
@@ -0,0 +1,7 @@
+# 3D-Unet Inference with AMD MIGraphX
+
+This example provides how to apply image segmentation to 3D images using AMD MIGraphX on a given AMD GPU. 
+
+## How to:
+1) User will need to have access to BRATS dataset. 
```

Comment:
It would be helpful to give more detailed instructions for how to access the dataset.

Reply:
added the link for getting access